cluster-http/src/main/scala/akka/management/cluster/ClusterHealthCheck.scala

```diff
@@ -1,17 +1,3 @@
 /*
  * Copyright (C) 2017-2018 Lightbend Inc. <http://www.lightbend.com>
  */
```

Comment:
Looks like this somehow ended up as an otherwise empty file with a copyright header.

Reply:
Doh!